tyk-control-plane/values.yaml

```diff
@@ -763,6 +763,11 @@ tyk-bootstrap:
       name: Default Org
       # The hostname to bind the Portal to.
       cname: tyk-portal.local
+      # hybrid configures whether hybrid organisation is enabled during bootstrap or not.
+      hybrid:
+        # Enabling hybrid allows a worker gateways to login as an organisation member into MDCB.
+        # Set this field to true while using MDCB control plane.
+        enabled: false
```

Comment:
I think we should set this to true by default for control plane

Reply:
yes you are right. updated accordingly